gnovm/pkg/packages/imports.go

```diff
@@ -50,44 +50,57 @@ func Imports(pkg *std.MemPackage, fset *token.FileSet) (ImportsMap, error) {
 	return res, nil
 }
 
-// FileImport represents an import
-type FileImport struct {
-	PkgPath string
-	Spec    *ast.ImportSpec
-}
-
-// FileImports returns the list of gno imports in the given file src.
-// The given filename is only used when recording position information.
```

Comment:
Could `FileImports` docs be restored?

Reply:
8c463369cd66a758f829edee530f33c6cd1e0172